testsuite/unittests/test_dcgen_versus_I_e.sli

```diff
@@ -85,7 +90,8 @@ modeldict keys{
 	300 Simulate
 
 	% compare final membrane potentials
-	n1 GetStatus /V_m get n2 GetStatus /V_m get eq dup {(pass) =}{ (failed: unequal voltages) =  executive } ifelse res exch append /res Set
+	n1 GetStatus /V_m get n2 GetStatus /V_m get eq dup {(pass) =}{ (failed: unequal voltages) =  executive } ifelse 
+	res exch append /res Set
```

Comment:
Since it does not fit onto one line, I would suggest breaking it properly:
```python
n1 GetStatus /V_m get n2 GetStatus /V_m get eq dup 
  {(pass) =}
  { (failed: unequal voltages) =  executive } 
ifelse 
res exch append /res Set
```